go/gas-oracle/abis/OVM_GasPriceOracle.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
Make symbolic link to the contracts

Reply:
I'm not going to make this a symbolic link because changes could cause the program to not compile and the binding is committed into the repo and building it is not part of the build step so if this is a symlink then it can get out of date with the binding. Updating it should be explicit and the only time it should change in the near future is for https://github.com/ethereum-optimism/optimism/pull/1190